drag-and-drop/library/src/utils/runtime-flow-state.ts

```diff
@@ -628,6 +695,62 @@ export class RuntimeFlowState {
     );
   }
 
+  exportContent(): View {
+    const baseView = this.view;
+
+    /** Walks the drag and drop state to remove any drop target assets */
+    const removeDndStateFromView = (obj: unknown): any => {
```

Comment:
This might be better to define in a utility file or something since it is static and doesn't really need to be in the flow state manager.

Reply:
extracted exportContent  to a helper function